src/WebJobs.Script/Binding/ServiceBus/ServiceBusValidator.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Microsoft.ServiceBus;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public interface INamespaceManager
```

Comment:
It's only used to make this class (slightly) more unit testable.  I feel like in that context it makes sense to keep it in the file and not pollute the solution tree